### ISSUE TITLE
REDEEM - redeem whole balance sets incorrect value

### ISSUE THREAD

**Issue by rudolfix** [Neufund/platform-frontend#2560]
**Description**
My balance is 29.083350986799034666 but when I use redeem whole balance, the value is set to rounded up wallet balance which is of course more than real balance

**Environment**
Production - 0x64Be85331949105Eb0459B936abF474710B4601b

**Actual behaviour**
Rounded up wallet balance gets copied

**Expected behavior**
What is expected is that after `withdraw` function is called on `EuroToken` contract, the balance is exactly 0. This means that in my case we should withdraw exactly 29.083350986799034666 - so wallet balance with full amount.
We should however do not scare people with such high precision numbers so unfortunately when you use this option you should show **wallet balance rounded down** but withdraw actual high precision balance!
In other words - we display human readable string but withdraw actual full precision balance

Once we do it, we'll apply this behaviour in other places - investment flow, ETH withdraw (this is not part of this ticket)

**Screenshots**
![image.png](https://images.zenhubusercontent.com/5a72da804b5806bc2bc52ecc/f2cd985e-9e93-479c-88c8-9f41b554d443)

**Comment by glupiawymowka** [comment]
*tested on .io*
